backend/app.js

```diff
@@ -15,6 +18,12 @@ app.use((req, res) => {
   });
 });
 
+app.use((err, req, res, next) => {
+  const { status = 500, message = "Server error" } = err;
+  console.log(message);
+  res.status(status).json({ message });
+});
+
```

Comment:
I see what you want to do here. This is alright, though I suggest moving this handler into the middleware folder, calling it for example handleError. Also since the next variable isn't used, please remove it.

Reply:
There is some interesting behavior when I remove next variable, I'm getting not json, but HTML.
I think that next variable is necessary 

<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="utf-8">
    <title>Error</title>
</head>

<body>
    <pre>Error: Not authorized<br> &nbsp; &nbsp;at HttpError (file:///C:/Projects/electronic-voting-project/backend/helpers/HttpError.js:2:17)<br> &nbsp; &nbsp;at isAuthenticated (file:///C:/Projects/electronic-voting-project/backend/middlewares/isAuthenticated.js:8:19)<br> &nbsp; &nbsp;at Layer.handle [as handle_request] (C:\Projects\electronic-voting-project\backend\node_modules\express\lib\router\layer.js:95:5)<br> &nbsp; &nbsp;at next (C:\Projects\electronic-voting-project\backend\node_modules\express\lib\router\route.js:149:13)<br> &nbsp; &nbsp;at Route.dispatch (C:\Projects\electronic-voting-project\backend\node_modules\express\lib\router\route.js:119:3)<br> &nbsp; &nbsp;at Layer.handle [as handle_request] (C:\Projects\electronic-voting-project\backend\node_modules\express\lib\router\layer.js:95:5)<br> &nbsp; &nbsp;at C:\Projects\electronic-voting-project\backend\node_modules\express\lib\router\index.js:284:15<br> &nbsp; &nbsp;at Function.process_params (C:\Projects\electronic-voting-project\backend\node_modules\express\lib\router\index.js:346:12)<br> &nbsp; &nbsp;at next (C:\Projects\electronic-voting-project\backend\node_modules\express\lib\router\index.js:280:10)<br> &nbsp; &nbsp;at Function.handle (C:\Projects\electronic-voting-project\backend\node_modules\express\lib\router\index.js:175:3)</pre>
</body>

</html>

Here is html that i get when remove next